srcpkgs/python3-selectors2/template

```diff
@@ -1,12 +1,12 @@
-# Template file for 'python-selectors2'
-pkgname=python-selectors2
+# Template file for 'python3-selectors2'
```

Comment:
selectors2 is back-ported version of selectors for python 2.x and 3.{0,4}
selectors is included in Core Python from python 3.5, drop it

Reply:
And also drop selectors dependencies in offlineimap.
I don't use offlineimap, I couldn't test it, though.

Reply:
Currently offlineimap isn’t compatible with py3 but I think pbui is going to look at it; reason it is WIP atm